src/test/java/guitests/SortTest.java

```diff
@@ -50,5 +50,19 @@ public void sortTest () {
         press(KeyCode.SHIFT).press(KeyCode.SEMICOLON).release(KeyCode.SEMICOLON).release(KeyCode.SHIFT);
         type("-date");
         push(KeyCode.ENTER);
+        // Issue state (open)
+        doubleClick("#dummy/dummy_col0_filterTextField");
+        doubleClick("#dummy/dummy_col0_filterTextField");
```

Comment:
Hmm no. But this file contains this double doubleclick code all over the place. Lets fix it in #966?